.github/workflows/eerelease.yml

```
@@ -95,20 +95,19 @@ jobs:
           podman images
 
       - name: Login to the Github Container registry
-        env:
-          REGISTRY_USER: ${{ github.actor }}
         uses: redhat-actions/podman-login@v1
         with:
           registry: ghcr.io/${{ github.repository_owner }}
           username: ${{ github.actor }}
-          password: ${{ github.token }}
+          password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Upload artifact EE to ghcr
         uses: redhat-actions/push-to-registry@v2
         with:
           image: ${{ inputs.ee-name }}
           tags: ${{ env.EE_VERSION }}
           registry: ghcr.io/ansible-community
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Here username is missing, I would add it to be on the safe side:
```suggestion
          username: ${{ github.actor }}
          password: ${{ secrets.GITHUB_TOKEN }}
```

---

Updated.